src/js/hca/containers/ConfirmationPage.jsx

```diff
@@ -33,14 +33,22 @@ class ConfirmationPage extends React.Component {
           <h4 className="success-copy">You have successfully submitted your application for health care.</h4>
         </div>
         <div>
-          <p>We are processing your application. The Department of Veterans Affairs will contact you when we finish our review.</p>
+          <p>
+            We are processing your application.<br/>
+            The Department of Veterans Affairs will contact you when we finish our review.<br/>
+            <i>Please print this page for your records.</i>
+          </p>
           {time &&
             <div className="success-alert-box">
               <p className="success-copy">Form submitted: {moment(time).format('MMMM D, YYYY, h:mm a')}</p>
             </div>
           }
-          <p>Please print this page for your records.</p>
-          <p>If you have questions, call 1-877-222-VETS (8387) and press 2.</p>
```

Comment:
This has been moved up to be consistent with the mock on the original issue (which is for an education form). Please let me know if this should be moved back.